io.gitlab.JBeaumont81.Etesync-Knotes.yaml

```diff
@@ -0,0 +1,56 @@
+app-id: io.gitlab.JBeaumont81.Etesync-Knotes
+runtime: org.kde.Platform
+runtime-version: '6.9'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: etesync-knotes
+separate-locales: false
+rename-desktop-file: etesync-knotes.desktop
+rename-icon: etesync-knotes
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.kde.kwalletd6
```

Comment:
does it only support kwallet? this won't work on anything that uses `org.freedesktop.secrets`.